src/modules/NfsServer.rb

```diff
@@ -277,28 +278,28 @@ def Write
         end
 
         if @nfs_security
-          if !Service.active?("svcgssd")
-            unless Service.Start("svcgssd")
-              # FIXME svcgssd is gone! (only nfsserver is left)
+          if !Service.active?(GSSERVICE)
+            unless Service.Start(GSSERVICE)
+              # FIXME #{GSSERVICE} is gone! (only nfsserver is left)
               Report.Error(
                 _(
-                  "Unable to start svcgssd. Ensure your kerberos and gssapi (nfs-utils) setup is correct."
+                  "Unable to start #{GSSERVICE}. Ensure your kerberos and gssapi (nfs-utils) setup is correct."
```

Comment:
We cannot use interpolation in translations.

See [Travis complain](https://travis-ci.org/yast/yast-nfs-server/builds/637522222#L166)

> Note: #{foo} substitution in translatable strings does not work properly, use
> 
>  _("foo %{bar} baz") % { :bar => bar }
>
> or
> 
>  _("foo %s baz") % bar

and also the explanation [at YaST Documentation site](https://yastgithubio.readthedocs.io/en/latest/localization/#interpolations).